### PR TITLE
s/budget/wallet/; s/allocate|allocation/budget/;

### PR DESCRIPTION
**Pull request by cmars**

### CODE REVIEW

File: wireformat/budget/entities.go [hunk 1]
```diff
@@ -9,43 +9,42 @@ import (
 	"strings"
 )
 
-// BudgetWithAllocations represents the current state of the budget and its allocations.
-type BudgetWithAllocations struct {
-	Limit       string       `json:"limit, omitempty"`
-	Total       BudgetTotals `json:"total"`
-	Allocations []Allocation `json:"allocations, omitempty"`
+// WalletWithBudgets represents the current state of the budget and its allocations.
```
Comment (alesstimec) [inline wireformat/budget/entities.go]: represents the current state of the wallet and it's budgets..

File: wireformat/budget/entities.go [hunk 2]
```diff
@@ -54,40 +53,34 @@ type Allocation struct {
...
-	Consumed string `json:"consumed"`
-}
-
-// ListBudgetsResponse is returned by the ListBdugets API call.
-type ListBudgetsResponse struct {
-	Budgets BudgetSummaries `json:"budgets, omitempty"`
-	Total   BudgetTotals    `json:"total, omitempty"`
+// ListWalletsResponse is returned by the ListBdugets API call.
+type ListWalletsResponse struct {
+	Wallets WalletSummaries `json:"budgets, omitempty"`
```
Comment (alesstimec) [inline wireformat/budget/entities.go]: do json tags need to change as well?

Comment (cmars) [inline wireformat/budget/entities.go]: Yes, thanks for catching this!

**Comment by cmars** [comment]
$$merge$$

**Pull request merged**